From 8e53fcc759a6096e92ccf4761ad7164abcda90c7 Mon Sep 17 00:00:00 2001
From: Bharat Bhushan <bbhushan2@marvell.com>
Date: Tue, 4 May 2021 11:24:43 +0530
Subject: [PATCH 1515/1921] dt-bindings: perf: marvell: add cn10k ddr
 performance monitor

Add binding documentation for the Marvell CN10k DDR
performance monitor unit.

Change-Id: I3b895da986bae99e65b58a7ded7f1cb70026ffd8
Signed-off-by: Bharat Bhushan <bbhushan2@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/51043
Reviewed-by: Sunil Kovvuri Goutham <sgoutham@marvell.com>
Tested-by: Sunil Kovvuri Goutham <sgoutham@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 .../bindings/perf/marvell-cn10k-ddr.txt           | 15 +++++++++++++++
 1 file changed, 15 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/perf/marvell-cn10k-ddr.txt

diff --git a/Documentation/devicetree/bindings/perf/marvell-cn10k-ddr.txt b/Documentation/devicetree/bindings/perf/marvell-cn10k-ddr.txt
new file mode 100644
index 000000000000..a2a593e99963
--- /dev/null
+++ b/Documentation/devicetree/bindings/perf/marvell-cn10k-ddr.txt
@@ -0,0 +1,15 @@
+* Marvell CN10k DDR performance monitor
+
+Required properties:
+
+- compatible: should be one of:
+	"marvell,cn10k-ddr-pmu"
+
+- reg: physical address and size
+
+Example:
+
+	ddrcpmu {
+		compatible = "marvell,cn10k-ddr-pmu";
+		reg = <0x87e1 0xc0000000 0x0 0x10000>;
+	};
-- 
2.31.1

